citi-onboarding/DADe-onepage
# Feature/add organization responsivity

## What I did

- Made Organization View completely responsive (Added mobile support)

| Mockup | Real |
|--------|------|
| ![organization mockup mobile](https://user-images.githubusercontent.com/61971624/122957886-bf009000-d358-11eb-844e-7f4ef7ad3143.png) | ![organization mockup real](https://user-images.githubusercontent.com/61971624/122959777-81503700-d359-11eb-90c7-f6942f7ad838.png) |



## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/add-organization-responsivity`
   4. `git pull origin feature/add-organization-responsivity`
   5. `cd server/`
   6. `node index.js`
   7. Open a new terminal
   8. `cd client`
   9. `npm start` 

2. In your browser go to http://localhost:3000 (should be open already)

3. Check if the organization view looks just like the mockup in desktop screen sizes (max-width > 1050px)

4. Check if the contact view looks just like the mockup in mobile screen sizes (max-width < 1050px)

5. Use inspect element menu and see how it looks on every device (remember that the mockup has a width of 414px). OBS: Maybe organization cards will be displayed as column instead of group of 2, but its fine.

6. Read the changed files section on GitHub 

7. Test the things that you think are worth testing, even the ones that are not in this description
